packages/nimble-components/src/table-column/enum-base/models/column-validator.ts

```diff
@@ -0,0 +1,50 @@
+import { nullableNumberConverter } from '@microsoft/fast-element';
+import type { Mapping } from '../../../mapping/base';
+
+/**
+ * Helper to share logic between the enum-text and icon column validators
+ */
+export class TableColumnEnumValidationHelper {
+    public static invalidMappingKeyValueForType(
+        keys: unknown[],
+        keyType: string
+    ): boolean {
+        let invalid = false;
+        if (keyType === 'number') {
+            invalid = keys.some(
+                x => x !== undefined
+                    && nullableNumberConverter.fromView(x) === null
+            );
+        } else if (keyType === 'boolean') {
+            // never invalid
```

Comment:
Well, I'm using the FAST `booleanConverter` which converts everything that isn't one of a few falsy values to `true`: https://github.com/microsoft/fast/blob/50e3fcd207adca50c5e666b12fc7c72b3cc0ba07/packages/web-components/fast-element/src/components/attributes.ts#L82

I guess I could use my own logic instead. I would just have to be sure to handle both boolean and string values. I'm kind of ambivalent about whether FAST's converter is good enough or not. Thoughts?

Reply:
That seems reasonable to me. I think it would be helpful to extend the comment here a bit to briefly explain why boolean keys are never invalid (i.e. that their values are coerced through a converter to always be `true` or `false`).

Reply:
I'm more on the fence. This allows you to leave off the key altogether and have it imply a value of false and a key of "FALSE" and have it mean true. I think I'd rather go strict in this case, key must be either the string "false" or the string "true" or the state is invalid.

Reply:
Made the validation stricter.